tests/drivers/build_all/i3c/boards/qemu_cortex_m3.overlay

```diff
@@ -16,4 +16,16 @@
 		i3c-scl-hz = <12500000>;
 		i2c-scl-hz = <400000>;
 	};
+
+	i3c1: i3c@cccccccc {
+		compatible = "snps,designware-i3c";
+		#address-cells = <3>;
+		#size-cells = <0>;
+		reg = <0xcccccccc 0x400>;
+		interrupt-parent = <&nvic>;
+		interrupts = <5 1>;
+		input-clock-frequency = <200000000>;
+		i3c-scl-hz = <12500000>;
+		i2c-scl-hz = <400000>;
```

Comment:
where is this prop used in the driver?

Reply:
it's not...this IP is rather stringent with the specification and has seperate registers for it's FM+ and FM I2C rates

Reply:
if not used/supported, it should be removed.